Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/VisualProfile.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Theme scriptableObject for loading theme settings
```

Comment:
Is this correct? This class isn't a `ScriptableObject`

Reply:
No, it was a scriptableobject but I couldn't get the inspector behavior etc I wanted ...fixing